src/frontend/planner_test/tests/testdata/predicate_pushdown.yaml

```diff
@@ -56,9 +56,9 @@
         └─LogicalHopWindow { time_col: t.ts, slide: 1 day, size: 3 days, output: all }
           └─LogicalScan { table: t, columns: [t.v1, t.v2, t.v3, t.v4, t.ts, t._row_id] }
   optimized_logical_plan: |
-    LogicalFilter { predicate: (t.v1 = 10:Int32) AND (t.v2 = 20:Int32) AND (t.v3 = 30:Int32) AND (t.ts >= '1997-07-01':Varchar::Date) AND (window_start >= '1997-07-02':Varchar::Date) AND (window_end >= '1997-07-03':Varchar::Date) AND (window_start >= (t.ts + '1 day':Interval)) AND (window_end > (t.ts + '4 days':Interval)) }
+    LogicalFilter { predicate: (window_start >= '1997-07-02':Varchar::Date) AND (window_end >= '1997-07-03':Varchar::Date) AND (window_start >= (t.ts + '1 day':Interval)) AND (window_end > (t.ts + '4 days':Interval)) }
     └─LogicalHopWindow { time_col: t.ts, slide: 1 day, size: 3 days, output: all }
-      └─LogicalScan { table: t, columns: [t.v1, t.v2, t.v3, t.v4, t.ts] }
+      └─LogicalScan { table: t, columns: [t.v1, t.v2, t.v3, t.v4, t.ts], predicate: (t.v1 = 10:Int32) AND (t.v2 = 20:Int32) AND (t.v3 = 30:Int32) AND (t.ts >= '1997-07-01':Varchar::Date) }
```

Comment:
may be we can add a planner test which just get window_start or window_end column so that we can see if the `output_indices` is right. :thinking:

Reply:
Added below.